pkgs/development/tools/dep2nix/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, buildGoPackage
+, makeWrapper, nix-prefetch-git }:
+
+buildGoPackage rec {
+  name = "dep2nix-${version}";
+  version = "0.0.1";
+
+  goPackagePath = "github.com/nixcloud.io/dep2nix";
```

Comment:
Please fix this too.

Reply:
Also fixed upstream: https://github.com/nixcloud/dep2nix/pull/11